images/installer/nested-libvirt/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# This Dockerfile is a used by CI to test a libvirt cluster launched in a gce instance
+# It builds an image containing google-cloud-sdk, ns_wrapper and scripts to launch a VM for a libvirt install.
+FROM openshift/origin-release:golang-1.10 AS build
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+RUN hack/build.sh && hack/get-terraform.sh
+
+FROM centos:7
+COPY --from=build /go/src/github.com/openshift/installer/images/installer/nested-libvirt/google-cloud-sdk.repo /etc/yum.repos.d/google-cloud-sdk.repo
+COPY --from=build /go/src/github.com/openshift/installer/bin/openshift-install /bin/openshift-install
+COPY --from=build /go/src/github.com/openshift/installer/bin/terraform /bin/terraform
+COPY --from=build /go/src/github.com/openshift/installer/scripts/ci/router-check.sh /bin/router-check.sh
+COPY --from=build go/src/github.com/openshift/installer/images/installer/nested-libvirt/mock-nss.sh /bin/mock-nss.sh
```

Comment:
nit: Missing a leading `/` here vs. your other `COPY` lines.

Also, could we alphabetize these `COPY` lines?  That would get the two `.../bin/...` entries first, allowing us to share build-cache with `images/installer/Dockerfile.ci`.

It would be nice if there was a way to explicitly extend `images/installer/Dockerfile.ci` here (much, much more on that in moby/moby#735), but for now repeating the same lines in each Dockerfile seems easier than asking users to build `images/installer/Dockerfile.ci` to a particular local tag and then `FROM that-tag` here.

Reply:
done